index.html

```diff
@@ -129,7 +129,6 @@ <h3>Concept and Initial Report</h3>
           type="range"
           step="0.5"
           min="0"
-          value="0"
```

Comment:
This is going to be set in draw().
Also by setting it early, it makes debugging with developer tools a little harder to read.